README.md

```diff
@@ -1,2 +1,2 @@
-# silverash-mythicplus-bot
+# Silverash MythicPlus Bot
 M+ bot for Silverash
```

Comment:
nit: change the description to be a little bit more detailed as well

Reply:
```suggestion
M+ bot for SilverASS
```